pkg/healthchecker/health_checker.go

```diff
@@ -102,10 +96,22 @@ func (hc *healthChecker) CheckHealth() (bool, error) {
 
 // logPatternHealthCheck checks for the provided logPattern occurrences in the service logs.
 // Returns true if the pattern is empty or does not exist logThresholdCount times since start of service, false otherwise.
-func logPatternHealthCheck(service, logStartTime string, logPatternsToCheck map[string]int) (bool, error) {
+func logPatternHealthCheck(service string, loopBackTime time.Duration, logPatternsToCheck map[string]int) (bool, error) {
```

Comment:
Will this function always return in the windows case ? len(logPatternsToCheck) == 0 -> true ?

Reply:
Not sure. As of now, currently with our windows health checkers we don't have any patterns to check, but that's not to say that it may be different for future problem detectors